README.md

```diff
@@ -7,6 +7,10 @@ products:
 - azure
 ---
 
+# IMPORTANT!
+
+This repository is a fork. Changes concerned mainly azure-migrate/automation-script

```

Comment:
This repo is not a fork. Does the source for these samples exist somewhere else?